src/app/users/users-billing-info/users-billing-info.component.ts

```diff
@@ -10,13 +15,45 @@ import { SharedService } from '../../store/services';
 })
 export class UsersBillingInfoComponent implements OnDestroy, OnInit {
 
-  constructor(private sharedService: SharedService) { }
-
+  constructor(private sharedService: SharedService, private store: Store<AuthState>, private formBuilder: FormBuilder) { }
+  private cardNumber;
+  billingForm: FormGroup;
+  private expiryMonth = 'Month';
+  private expiryYear = 'Year';
+  private cvc;
+  private months = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'];
+  private years = ['2018', '2019', '2020', '2021', '2022', '2023', '2024', '2025', '2026'];
   ngOnInit() {
     this.sharedService.hideFooter.emit(true);
+    setTimeout(() => this.store.dispatch(new FinalLoading({ loadingState: false })));
+
+    this.billingForm = this.formBuilder.group({
+      'cardNumber': ['', [Validators.minLength(16), Validators.maxLength(16)]]
+    });
   }
 
+  getToken() {
+    console.log(this.cardNumber, this.expiryMonth, this.expiryYear, this.cvc);
+    (<any>window).Stripe.card.createToken({
+      number: this.cardNumber,
+      exp_month: this.expiryMonth,
+      exp_year: this.expiryYear,
+      cvc: this.cvc
+    }, (status: number, response: any) => {
+      if (status === 200) {
+        console.log(`Success! Card token ${response.card.id}.`);
```

Comment:
Please remove console.log before pushing changes, if you want to do something later, then you can add a `TODO `